extensions/2.0/Khronos/KHR_texture_astc/README.md

```diff
@@ -0,0 +1,253 @@
+<!--
+Copyright 2024 The Khronos Group Inc.
+SPDX-License-Identifier: LicenseRef-KhronosSpecCopyright
+-->
+
+# KHR_texture_astc
+
+## Contributors
+
+- Wasim Abbas [@wasimabbas-arm](https://github.com/wasimabbas-arm)
+- Jan-Harald Fredriksen [@janharaldfredriksen-arm](https://github.com/janharaldfredriksen-arm)
+- Peter Harris [@solidpixel](https://github.com/solidpixel)
+- Mark Callow [@MarkCallow](https://github.com/MarkCallow)
+
+Copyright 2023 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds the ability to specify textures using KTX v2 images with (ASTC compression)[https://github.com/ARM-software/astc-encoder/blob/main/Docs/FormatOverview.md]. An implementation of this extension can use such images as an alternative to the PNG or JPEG images available in glTF 2.0 for more efficient asset transmission, reducing GPU memory footprint, higher texture throughput and faster execution.
```

Comment:
"using ASTC compressed images in KTX v2 containers" would be more accurate.

Reply:
Fixed in https://github.com/KhronosGroup/glTF/pull/2369/commits/b7b6dd2397b7df15f740e40265650c74abb0cb77